modules/ROOT/pages/op-installing_pipelines.adoc

```diff
@@ -0,0 +1,25 @@
+[id="op-installing-openshift-pipelines"]
+= Installing OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: installing-openshift-pipelines
+
+toc::[]
+
+// Installing Openshift Pipelines Operator
+[id="installing-pipelines-operator_{context}"]
+== Installing the {op-title} Operator
+
+The {op-title} Operator can be installed using the {product-title} instructions for installing Operators.
+
+You can install the {op-title} Operator in the host cluster by following the {product-title} instructions on installing an Operator.
+
+For more details on installation instructions, see the {product-title} documentation on xref:../operators/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[adding Operators to a cluster].
+
```

Comment:
The 3 sentences on L 13, 15 and 17 say the same thing, please combine them into one with the link.